apisix/core/ctx.lua

```diff
@@ -206,6 +206,7 @@ do
     local apisix_var_names = {
         balancer_ip = true,
         balancer_port = true,
+        consumer_group_id = true,
```

Comment:
Need to update https://github.com/apache/apisix/blob/master/docs/en/latest/apisix-variable.md for the new variable.